py/server/deephaven/updateby.py

```diff
@@ -164,6 +171,211 @@ def ema_time_decay(ts_col: str, time_scale: Union[int, str], cols: Union[str, Li
         raise DHError(e, "failed to create a time-decay EMA UpdateByOperation.") from e
 
 
+def ems_tick_decay(time_scale_ticks: int, cols: Union[str, List[str]],
```

Comment:
These new methods suffer from the same problem as the ticket I have reported twice.  The timescalee should *not* be int.  It should be float.

I wasn't able to find the other ticket to link to.  Maybe it was fixed?

Reply:
Here are the already reported issues: 
#2660 
#3670

Reply:
This is being addressed in #3745, to merge after this.